src/engine.jl

```diff
@@ -40,53 +36,49 @@ type MSession
             # Hide the MATLAB Command Window on Windows
             ccall(engfunc(:engSetVisible ), Cint, (Ptr{Void}, Cint), ep, 0)
         end
-
-        println("A MATLAB session is open successfully")
         new(ep, buf, bufptr)
     end
-
-    MSession() = MSession(default_output_buffer_size)
 end
 
+
 function close(session::MSession)
     # Close a MATLAB Engine session
-
     @assert libeng::Ptr{Void} != C_NULL
-
     r = ccall(engfunc(:engClose), Cint, (Ptr{Void},), session.ptr)
```

Comment:
Should I just revert back to what I have?

Are there any other changes you need, before green lighting this PR?